package.json

```diff
@@ -115,20 +115,20 @@
   },
   "keywords": [],
   "dependencies": {
-    "@angular/animations": "^10.2.2",
-    "@angular/cdk": "^10.2.2",
-    "@angular/common": "^10.2.2",
-    "@angular/compiler": "^10.2.2",
-    "@angular/core": "^10.2.2",
-    "@angular/elements": "10.2.2",
+    "@angular/animations": "^11.2.10",
+    "@angular/cdk": "^11.2.10",
+    "@angular/common": "^11.2.10",
+    "@angular/compiler": "^11.2.10",
+    "@angular/core": "^11.2.10",
+    "@angular/elements": "11.2.10",
```

Comment:
This update is required because of a bug in @angular/elements that was fixed in this version `11.2.10`. https://github.com/angular/angular/issues/25525

@vojtechjelinek

Reply:
Yep, I'm always happy when we are upgrading :smile: